src/ui/public/vislib/visualizations/_map.js

```diff
@@ -17,12 +17,13 @@ export default function MapFactory(Private, tilemap) {
   let defaultMapCenter = [15, 5];
   let defaultMarkerType = 'Scaled Circle Markers';
 
+  let tilemapOptions = tilemap.options;
+
+  tilemapOptions.attribution = marked(tilemapOptions.attribution);
+
   let mapTiles = {
     url: tilemap.url,
-    options: {
-      attribution: marked(tilemap.attribution),
-      subdomains: tilemap.subdomains
-    }
+    options: tilemapOptions 
```

Comment:
There is a whitespace character at the end of this line that is somehow breaking tests.